### PR TITLE
Added documentation on the template API for custom sms messages

### PR DESCRIPTION
**Pull request by nikolaystoilov-okta**
Updated the usage in the factors API.
Added postman scripts to validate the new API calls.

Resolves: OKTA-93212

Bacon: test

### CODE REVIEW

**Comment by nikolaystoilov-okta** [comment]
Please review @mystiberry-okta , @karlmcguinness-okta , @slee-okta , @joelfranusic-okta , @tthompson-okta @srinivasanagandla-okta @authteam-okta Thank you

**Comment by mystiberry-okta** [comment]
Looks great! I'm going to do some simple copy edits for style, but the content looks great to me! Thank you!

File: _source/docs/api/resources/templates.md [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+
+Adds a new custom sms template to your organization.
+
+##### Request Parameters
+{:.api .api-request .api-request-params}
+
+Parameter | Description                               | ParamType | DataType                          | Required | Default
+--------- | ----------------------------------------- | --------- | --------------------------------- | -------- | ---
+          | Definition of the new custom sms template | Body      | (#template-object)                | TRUE     |
+
```
Comment (mystiberry-okta) [inline _source/docs/api/resources/templates.md]: Is the parameter name missing here?

File: _source/docs/api/resources/templates.md [hunk 2]
```diff
@@ -0,0 +1,453 @@
...
+### Remove Sms Template
+{:.api .api-operation}
+
+<span class="api-uri-template api-uri-delete"><span class="api-label">DELETE</span> /templates/sms/*:id*</span>
+
+Removes a sms template.
+
+> The default sms template cannot be removed.
+
```
Comment (mystiberry-okta) [inline _source/docs/api/resources/templates.md]: What happens if you remove an SMS template that's referenced somewhere else in the system? Does an error occur, or are you not allowed to delete it?

**Comment by mystiberry-okta** [comment]
Style edits complete. I did have two questions after all on the first commit.

File: _source/docs/api/resources/templates.md (hunk 2 above)
Comment (nikolaystoilov-okta) [inline _source/docs/api/resources/templates.md]: @mystiberry-okta It is the same as if you are trying to use a templateId that doesn't exists. We will fail back to the default template.

File: _source/docs/api/resources/templates.md (hunk 1 above)
Comment (nikolaystoilov-okta) [inline _source/docs/api/resources/templates.md]: @mystiberry-okta No, it is not missing. When we create a new template we just send a new object in json format in the body.

**Comment by mystiberry-okta** [comment]
Can someone let me know when the tech review is finished? Assuming I can merge and push for next Wed. Also assuming we should put Custom SMS in the release notes now, @nikolaystoilov-okta ?

**Pull request merged**